src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.ts

```diff
@@ -126,7 +126,7 @@ export class EditDatumGroepComponent extends EditComponent implements OnInit {
     }
 
     edit(editing: boolean): void {
-        if (!this.readonly && !this.utilService.hasEditOverlay()) {
+        if (!this.readonly && !this.utilService.hasEditOverlay() && this.isWijzigbaar) {
```

Comment:
!this.readonly en his.isWijzigbaar lijken zo op het eerste gezicht dezelfde betekenis te hebben. Zijn ze allebei nodig?